### ISSUE TITLE
Offline mode

### ISSUE THREAD

**Issue by galenriley** [Chattahooligans/hooligan-hymnal-app#35]

**Comment by galenriley** [comment]
Add some sort of indicator across the UI when we could not hit the server and may not have the latest data. Something like a no connection banner

**Comment by tacoman** [comment]
This can reasonably be implemented per-area, e.g. roster, songbook, feed, etc. On any successful load, we can store the returned JSON in persistent storage. Any attempt to load data should try the network first, and if that fails return the stored data. The result should also include a property indicating whether the data came from online or offline, and control the presence of the banner accordingly.

**Comment by galenriley** [comment]
Agreed. I'd like to see this alongside a pull-down-to-refresh gesture on the roster screen and songbook ToC
